modules/op-creating-mounting-kms-authentication-token-secret.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-mounting-kms-authentication-token-secret_{context}"]
+= Creating and mounting the KMS authentication token secret
+
+You can provide the authentication token for the KMS server (for example, the the value of `VAULT_TOKEN` if the KMS provider is Hashicorp Vault) using a secret. You must create this secret, mount it on the {tekton-chains} controller, and set the `signers.kms.auth.token-dir` parameter to the directory where the secret is mounted.
```

Comment:
This has been changed now, user can create a secret with any key, and `VAULT_TOKEN`  is not mandatory now https://github.com/tektoncd/chains/blob/main/docs/config.md#kms-oidc-and-spire-configuration. Also the field is changed from `signers.kms.auth.token-dir` to `signers.kms.auth.token-path`

And this is the related PR https://github.com/tektoncd/chains/pull/1167

Reply:
@PuneetPunamiya please clarify. Does this mean that the `signers.kms.auth.token-path` parameter must specify the full path and name of the file to read? Or does it still specify a directory but any file from that directory is read?

If one follows the examples currently in the doc, the token is in the file named `/etc/kms-secrets/KMS_AUTH_TOKEN` . In this example, should `signers.kms.auth.token-path` be set to `/etc/kms-secrets` or `/etc/kms-secrets/KMS_AUTH_TOKEN` ?

Reply:
Yeah the parameter means that now it specifies the full path and name of the file to read.

In the example you mentioned above if `signers.kms.auth.token-path` is specified then it's value should be `/etc/kms-secrets/KMS_AUTH_TOKEN`